packages/store-sync/src/postgres-decoded/buildColumn.ts

```diff
@@ -14,12 +14,12 @@ export function buildColumn(name: string, schemaAbiType: SchemaAbiType) {
     case "int8":
     case "int16":
     case "uint24":
-    case "uint32":
     case "int24":
     case "int32":
       // integer = 4 bytes (https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-INT)
       return asNumber(name, "integer");
 
+    case "uint32":
     case "uint40":
     case "uint48":
     case "int40":
```

Comment:
can't comment below but does `int48` need to be bumped to the next size up too?

Reply:
> can't comment below 

What does that mean sorry? Anyway I don't think so, int48 is already a postgres `bigint` here?

Reply:
We chatted IRL and we mixed up the JavaScript and Postgres column types. 

but maybe `uint64` and `int64` need to move down because they may not fit within the `bigint` column type

Reply:
@holic it definitely doesn't fit - `uint64` is 8 bytes unsigned, postgres `bigint` is 8 bytes signed:

https://www.postgresql.org/docs/current/datatype-numeric.html